src/vm-repair/azext_vm_repair/tests/latest/test_repair_commands.py

```diff
@@ -0,0 +1,40 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer
```

Comment:
Would you had linux tests as well? Not necessry now but would be really helpful in the future

Reply:
Will add tests for linux today.